tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/ui/wizard/newapp/ResourceDetailsContentProviderTest.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.test.ui.wizard.newapp;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+
+import org.jboss.tools.openshift.internal.ui.wizard.newapp.ResourceDetailsContentProvider;
+import org.jboss.tools.openshift.internal.ui.wizard.newapp.ResourceDetailsContentProvider.ResourceProperty;
+import org.jboss.tools.openshift.test.util.ResourceMocks;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.openshift.restclient.model.IBuildConfig;
+
+/**
+ * @author Andre Dietisheim
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class ResourceDetailsContentProviderTest {
+
+	private ResourceDetailsContentProvider contentProvider;
+
+	@Before
+	public void before() {
+		this.contentProvider = new ResourceDetailsContentProvider();
+	}
+
+	@Test
+	public void shouldReturnUnknownResourcePropertyGivenBuildConfigBuildßStrategyIsNull() {
```

Comment:
`...BuildßStrategy...`
@adietish scharfes S hid in the method name. Could you, please, remove it and we can merge it